### PR TITLE
Doc/release notes 0.2

### LINKED ISSUE #112: Roadmap for Release 0.2
**Problem description**
As decided during the last meeting, we are opening this issue to summarize the decisions and to be used as reference in the PRs.

Release 0.2.0 of the DeviceLocation API family will contain:

* API spec for location-verification, v0.2.0
* API spec for location-retrieval, v0.1.0
* API spec for geofencing, v0.1.0
* Updated README and CHANGELOG

**Expected action**

* Telefónica to create a Release Candidate version of location-verification v0.2.0-rc @jlurien 
* Orange to create a Release Candidate version of location-retrieval v0.1.0-rc @bigludo7 
* DT to create a Release Candidate version of geofencing v0.1.0-rc @akoshunyadi 

taking into consideration:

* info.description updated with latest documentation
* scopes and securitySchemes aligned
* review examples
* assure that spec is correctly displayed in Swagger Editor and Redocly

### PR DESCRIPTION
**Pull request by jlurien**
#### What type of PR is this?

* documentation

#### What this PR does / why we need it:

In preparation for Release 0.2:

- README updated
- New CHANGELOG with changes from 0.1 to 0.2

#### Special notes for reviewers:

Links pointing to release-0.2.0 branch will not work until the branch is created with this PR is merged.

#### Fixes

#112

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,6 +24,26 @@ Repository to describe, develop, document and test the DeviceLocation API family
...
+    - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/DeviceLocation/blob/release-0.2.0/code/API_definitions/location-retrieval.yaml)
+  - **geofencing v0.1.0-wip**, which is still a pre-alpha release.
+    - OpenAPI [YAML spec file](https://github.com/camaraproject/DeviceLocation/blob/release-0.2.0/code/API_definitions/geofencing.yaml)
+    - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/DeviceLocation/blob/release-0.2.0/code/API_definitions/geofencing.yaml&nocors)
+    - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/DeviceLocation/blob/release-0.2.0/code/API_definitions/geofencing.yaml)
+
+* The previous release version v0.1.0 of DeviceLocation API is available within the [release-0.1.0 branch](https://github.com/camaraproject/DeviceLocation/tree/release-0.1.0)
+  - This past release only included the first alpha version of the API now renamed to location-verification, but it was then named as "location v0.1.0"
+
 ## Contributorship and mailing list
```
Comment (rartych) [inline README.md:47]: Should we add here the link to the channel in CAMARA Slack? - https://camara-project.slack.com/archives/C062ZUVQH1N

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+- **This release contains significant breaking changes compared to v0.1.0, and it is not backward compatible**
+  - Especially a lot of the parameter names changed in line with the agreed glossary within CAMARA Commonalities
+- This is an alpha version, it should be considered as a draft.
+- There are bug fixes to be expected and incompatible changes in upcoming versions. 
+- The release is suitable for implementors, but it is not recommended to use the API with customers in productive environments.
+
+
+## What's changed
+
+*  Second alpha version of `location-verification` API, v0.2.0, ranamed from `location`, with main changes:
```
Comment (sfnuser) [inline CHANGELOG.md:27]:
```suggestion
*  Second alpha version of `location-verification` API, v0.2.0, renamed from `location`, with main changes:
```

**Review by sfnuser — APPROVED** [review]
Looks good. Thanks.

File: CHANGELOG.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+*  Second alpha version of `location-verification` API, v0.2.0, renamed from `location`, with main changes:
+    - Input property `ueId` evolved to `device`, following CAMARA guidelines.
+    - New input parameter `area`, considering initially circular areas, and deprecating previous `latitude`, `longitude` and `accuracy` properties.
+    - New input parameter `maxAge`.
+    - Response adds new values `UNKNOWN` and `PARTIAL` for `verificationResult`. It also adds properties `matchRate` and `lastLocationTime`.
+    - Security schemas and scopes aligned with commonalities guidelines.
+    - Error responses adapted to new commonalities guidelines.
+    - Inline documentation within OpenAPI spec. 
+
+* Pre-alpha version of the new API `location-retrieval`, v0.1.0-wip
```
Comment (jlurien) [inline CHANGELOG.md:36]:
```suggestion
* First alpha version of the new API `location-retrieval`, v0.1.0-wip

```

File: CHANGELOG.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+    - Initial version already aligned with latest guidelines.
+    - Returns 2 possible location areas: `Circle` and `Polygon`.
+
+* Pre-alpha version of the new API `geofencing`, v0.1.0-wip
```
Comment (jlurien) [inline CHANGELOG.md:40]:
```suggestion
* First alpha version of the new API `geofencing`, v0.1.0-wip

```

File: CHANGELOG.md [hunk 5]
```diff
@@ -0,0 +1,62 @@
...
+- **This release contains significant breaking changes compared to v0.1.0, and it is not backward compatible**
+  - Especially a lot of the parameter names changed in line with the agreed glossary within CAMARA Commonalities
+- This is an alpha version, it should be considered as a draft.
+- There are bug fixes to be expected and incompatible changes in upcoming versions. 
+- The release is suitable for implementors, but it is not recommended to use the API with customers in productive environments.
+
+
+## What's changed
+
+*  Second alpha version of `location-verification` API, v0.2.0, renamed from `location`, with main changes:
```
Comment (jlurien) [inline CHANGELOG.md:27]:
```suggestion
* Second alpha version of `location-verification` API, v0.2.0, renamed from `location`, with main changes:

```

File: README.md [hunk 6]
```diff
@@ -11,7 +11,9 @@ Repository to describe, develop, document and test the DeviceLocation API family
 ## Scope
 * Service APIs for “Device Location” (see APIBacklog.md)  
 * It provides the customer with the ability to:  
-  * check the location of a device.
+  * verify the location of a device.
+  * retrieve the location of a device (WIP).
```
Comment (jlurien) [inline README.md:15]:
```suggestion
  * retrieve the location of a device.
```

File: README.md [hunk 7]
```diff
@@ -11,7 +11,9 @@ Repository to describe, develop, document and test the DeviceLocation API family
 ## Scope
 * Service APIs for “Device Location” (see APIBacklog.md)  
 * It provides the customer with the ability to:  
-  * check the location of a device.
+  * verify the location of a device.
+  * retrieve the location of a device.
+  * subscribe and receive notifications about a device entering or leaving certain location (geofencing) (WIP).
```
Comment (jlurien) [inline README.md:16]:
```suggestion
  * subscribe and receive notifications about a device entering or leaving certain location (geofencing). 
```

File: README.md [hunk 8]
```diff
@@ -21,6 +23,27 @@ Repository to describe, develop, document and test the DeviceLocation API family
...
+* Release 0.2.0 of the API family is available within the [release-0.2.0 branch](https://github.com/camaraproject/DeviceLocation/tree/release-0.2.0). The API family now includes 3 APIs, in different state of progress:
+  - **location-verification v0.2.0**, which is the second alpha release of this API.
+    - OpenAPI [YAML spec file](https://github.com/camaraproject/DeviceLocation/blob/release-0.2.0/code/API_definitions/location-verification.yaml)
+    - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/DeviceLocation/blob/release-0.2.0/code/API_definitions/location-verification.yaml&nocors)
+    - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/DeviceLocation/blob/release-0.2.0/code/API_definitions/location-verification.yaml)
+  - **location-retrieval v0.1.0-wip**, which is still a pre-alpha release.
```
Comment (jlurien) [inline README.md:36]:
```suggestion
  - **location-retrieval v0.1.0-wip**, which is the first alpha release.
```

File: README.md [hunk 9]
```diff
@@ -21,6 +23,27 @@ Repository to describe, develop, document and test the DeviceLocation API family
...
+    - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/DeviceLocation/blob/release-0.2.0/code/API_definitions/location-verification.yaml&nocors)
+    - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/DeviceLocation/blob/release-0.2.0/code/API_definitions/location-verification.yaml)
+  - **location-retrieval v0.1.0-wip**, which is the first alpha release.
+    - OpenAPI [YAML spec file](https://github.com/camaraproject/DeviceLocation/blob/release-0.2.0/code/API_definitions/location-retrieval.yaml)
+    - [View it on ReDoc](https://redocly.github.io/redoc/?url=https://raw.githubusercontent.com/camaraproject/DeviceLocation/blob/release-0.2.0/code/API_definitions/location-retrieval.yaml&nocors)
+    - [View it on Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/camaraproject/DeviceLocation/blob/release-0.2.0/code/API_definitions/location-retrieval.yaml)
+  - **geofencing v0.1.0-wip**, which is still a pre-alpha release.
```
Comment (jlurien) [inline README.md:40]:
```suggestion
  - **geofencing v0.1.0-wip**, which is the first alpha release.
```

**Review by bigludo7 — APPROVED** [review]
Look very good for me.
Thanks @jlurien

File: CHANGELOG.md [hunk 10]
```diff
@@ -0,0 +1,63 @@
...
+
+## What's changed
+
+* Second alpha version of `location-verification` API, v0.2.0, renamed from `location`, with main changes:
+
+    - Input property `ueId` evolved to `device`, following CAMARA guidelines.
+    - New input parameter `area`, considering initially circular areas, and deprecating previous `latitude`, `longitude` and `accuracy` properties.
+    - New input parameter `maxAge`.
+    - Response adds new values `UNKNOWN` and `PARTIAL` for `verificationResult`. It also adds properties `matchRate` and `lastLocationTime`.
+    - Security schemas and scopes aligned with commonalities guidelines.
```
Comment (sfnuser) [inline CHANGELOG.md:33]: @jlurien - Do we have an agreement on this topic in commonalities? I see [this](https://github.com/camaraproject/Commonalities/issues/53) issue still open?

Comment (jlurien) [reply CHANGELOG.md:33]: We still have to refine some details in Commonalities, but the most important applied change is the use of openIdConnect for securitySchemes, which was approved by the [TSC](https://github.com/camaraproject/Governance/blob/main/documentation/MeetingMinutes/TSC/TSC-2023-10-19-Meeting-Minutes.md#commonalities--icm). Hopefully we can advance on closing the other details before the Release Candidate is finally approved. If further changes come, we'll have to apply them in future releases.

**Comment by ABusyHippie** [comment]
@sfnuser or another reviewer. I'm with Centillion and was wondering if we could get access to the Alpha API? @iamdatanick mentioned this might be a good place to ask

**Comment by jlurien** [comment]
> @sfnuser or another reviewer. I'm with Centillion and was wondering if we could get access to the Alpha API? @iamdatanick mentioned this might be a good place to ask

Hi @ABusyHippie, not sure about what you mean by getting access to any Alpha API. What we call Alpha versions are the different versions of the specs, published and available in this Github repo. So far only Release 0.1.0 is available, in its own branch.

Anyway, if the question or comment is not directly related to this specific issue about the release notes, I would suggest to open a dedicated issue for it, so we can track it better.

**Comment by ABusyHippie** [comment]
Ok thank you for clarifying, I thought the 0.1.0 release may have some early stage endpoints for testing

**Pull request merged**